hail/python/hail/methods/misc.py

```diff
@@ -317,7 +317,7 @@ def filter_intervals(ds, intervals, keep=True) -> Union[Table, MatrixTable]:
 
     Notes
     -----
-    Based on the ``keep`` argument, this method will either restrict to points
+    Based on the `keep` argument, this method will either restrict to points
     in the supplied interval ranges, or remove all rows in those ranges.
 
     When ``keep=True``, partitions that don't overlap any supplied interval
```

Comment:
Should this be `keep=True`

Reply:
I think the rules we try to stick to are: \`\` for inline code snippets, \` for references to arguments or field names.

Reply:
Thanks, reading up on Sphinx rendering, I see that ``keep`` results in a code block (well, class='pre'). Thanks for explanation.